helm/alloy/examples/logs/values.yaml

```diff
@@ -44,3 +44,28 @@ alloy:
             url = "https://loki-gateway.loki/loki/api/v1/push"
           }
         }
+    mounts:
+      varlog: true
+      dockercontainers: true
+      extra:
+      - name: runlogjournal
+        mountPath: /run/log/journal
+        readOnly: true
+    securityContext:
+      runAsUser: 0
+      runAsGroup: 0
```

Comment:
Does it need to run as the root user? Also could we make sure this runs using a readonlyrootfilesystem?